core/frontend/src/render/RenderSystem.ts

```diff
@@ -600,12 +600,13 @@ export namespace RenderSystem { // eslint-disable-line no-redeclare
      */
     planProjections?: boolean;
 
-    /** To help prevent delays when a user interacts with a [[Viewport]], the WebGL render system precompiles shader programs before any Viewport is opened.
+    /** To help prevent delays when a user interacts with a [[Viewport]], the WebGL render system can precompile shader programs before any Viewport is opened.
      * This particularly helps applications when they do not open a Viewport immediately upon startup - for example, if the user is first expected to select an iModel and a view through the user interface.
      * Shader precompilation will cease once all shader programs have been compiled, or when a Viewport is opened (registered with the [[ViewManager]]).
-     * To disable this feature, set this to `false`.
+     * @note Enabling this feature can slow UI interactions before a [[Viewport]] is opened.
+     * To enable this feature, set this to `true`.
      *
-     * Default value: true
+     * Default value: false
      *
      * @public

```

Comment:
Change this back to `@beta`.
Alternatively, do `git revert 45e5b1717e96960881839fd491c1991deabe04d9` to get it back to the exact state it was before.

Reply:
Thanks; Changed back to beta.
I don't want to revert the entire previous PR because it fixed some issues with stopping precompiling if the system shut down in the middle of ongoing precompiling.